### PR TITLE
MP1 converted

### PR DESCRIPTION
**Pull request by ShanTulshi**
Addresses #23. Some rubric hiccups, but not sure how to resolve those without modifying the template so that will be in another issue.

### CODE REVIEW

**Comment by andrewipark** [comment]
I'm not sure what's going on in the rubric... for the elements with just "name", are you trying to make like rubric section headers?

**Comment by ShanTulshi** [comment]
@andrewipark yes, I would like nested rubric sections, for the different parts. Since that's not supported yet I just made blank rows with each MP section title

**Comment by andrewipark** [comment]
Why is there an empty section header for 1.1.2?

**Comment by ShanTulshi** [comment]
There's one for each section, isn't there?

File: _assignments/mp1.md [hunk 1]
```diff
@@ -4,38 +4,377 @@ title: "MP1: The Room"
...
+
+![Duplicate](/img/assignments/mp1/image4.png)
+
+From there, simply change the new plane’s rotation and position to make it one of the walls or ceilings. Unity measures position from the center of the object, so if you want your walls to match up with the floor (at height 0), your walls will need to be at 7.5
+
+![Scene window](/img/assignments/mp1/image15.png)
+
+Note: In the image above, rotating the plane also rotated its axes (the blue z axis now points down). Make sure to account for that when rotating and moving objects!
+
+#### Player:
```
[the remark this answers is not in the document]
Comment (ShanTulshi) [reply _assignments/mp1.md]: what, `Player`? Shouldn't it already be?

File: _assignments/mp1.md [hunk 2]
```diff
@@ -4,38 +4,377 @@ title: "MP1: The Room"
...
+
+*You will build a cubic room, whose sides are made out of six planes. 
+Make sure to orient these planes so the visible sides face inwards, and ensure that the player cannot walk through any of them. 
+The room should be 15x15x15 Unity units (aka meters).*
+
+First, create a plane. It can be found in the top bar menus, under `GameObject` → `3D Object`- > `Plane`, as shown below.
+
+![Create plane](/img/assignments/mp1/image3.png)
+
+Now, by default the plane is 10x10 (X x Z) units. In order to make your room 15 units wide, you have to scale the plane. On the right side (in the default editor layout) you will find the Inspector window. This window provides details about the currently selected object. Select the plane in the Scene view, and the Inspector will fill with information and settings for said plane. Find the “Scale” option, and set it to 1.5 on the X and Z to make your plane 15 units wide and long.
```
[the remark this answers is not in the document]
Comment (ShanTulshi) [reply _assignments/mp1.md]: Updated!

File: _assignments/mp1.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tojason) [reply _assignments/mp1.md]: screen shot 2019-01-16 at 10 52 09 am
The "Player" text is not bold. I want these section to have an obvious hierarchical. Or, maybe insert one more empty line before the ####Player.

File: _assignments/mp1.md [hunk 3]
```diff
@@ -4,38 +4,380 @@ title: "MP1: The Room"
...
+
+![Create plane](/img/assignments/mp1/image3.png)
+
+Now, by default the plane is 10x10 $$ (X \times Z) $$ units. In order to make your room 15 units wide, you have to scale the plane. On the right side (in the default editor layout) you will find the Inspector window. This window provides details about the currently selected object. Select the plane in the Scene view, and the Inspector will fill with information and settings for said plane. Find the “Scale” option, and set it to 1.5 on the $$ X $$ and $$ Z $$ to make your plane 15 units wide and long. 
+
+![Inspector](/img/assignments/mp1/image7.png)  
+Note: The plane has no thickness, so the value in $$ Y $$ can be any positive integer.
+
+By default, your scene has a directional light in it. This is basically the sun- a light source that illuminates your entire scene from a specified angle, from very far away. You’ll notice that your planes do not block this light. That’s because planes only block light (and render) from one side. Bear this in mind when creating objects in Unity in the future! Please delete the directional light (don’t worry, youll add new lights in later)
```
Comment (tojason) [inline _assignments/mp1.md]: missing a period.

Comment (tojason) [reply _assignments/mp1.md]: a period at the very end of this sentence.

File: _assignments/mp1.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShanTulshi) [reply _assignments/mp1.md]: It's the default h4 style, I tried making it bold and it looks less good IMO:
image

File: _assignments/mp1.md (hunk 3 above)
Comment (ShanTulshi) [reply _assignments/mp1.md]: fixed!

**Pull request merged**